SnapMD.VirtualCare.ApiModels/Rules/RuleEnums.cs

```diff
@@ -95,27 +95,27 @@ public enum RuleDecisionTypeCode
     public enum RuleConditionSource
     {
         /// <summary>
-        /// The unknown source
+        /// The metadata for creating rules.
         /// </summary>
-        Unknown,
+        Metadata,
```

Comment:
@jay-lee00 I like to have a meaningful "0" value for enums that would correspond to default(int) if the enum were not initialized, for example, "None".  For example, I've seen Country Code enums where Afghanistan was 0, and as a result the system thought everyone lived in Afghanistan. This is just an example of how a default value for an enum can lead to unexpected results.